vendor/engines/secure_rooms/app/controllers/secure_rooms_api/events_controller.rb

```diff
@@ -2,11 +2,37 @@ module SecureRoomsApi
 
   class EventsController < SecureRoomsApi::ApiController
 
+    before_action :parse_time
+
     def create
-      Rails.logger.info "SecureRoomsApi Event: #{params}"
+      SecureRooms::Alert.create(alert_params)
```

Comment:
would just using a `create!` and letting it end up in rollbar that way be sufficient?
@danhodos do you care about the response from this controller (if it's an error)?